docker/Dockerfile.testnet

```diff
@@ -0,0 +1,58 @@
+# Helper image to build renterd.
```

Comment:
It would be great if we could avoid adding another dockerfile that has almost 100% overlap with the production one. Instead I'd prefer injecting the `testnet` build tag through the `docker build` command.

Reply:
The `BUILD_TAGS` already work that way. I believe you can do `docker build --build-arg BUILD_TAGS='netgo testnet foo' ...`. I actually removed this file to deduplicate the logic here <https://github.com/SiaFoundation/renterd/pull/578>, so if anything we should update `hostd` perhaps?

Reply:
I removed the dockerfile.testnet. i think hostd or renterd need updated though. in both hostd/renterd the ghcr.io repos point to:

```
ghcr.io/siafoundation/renterd:latest
ghcr.io/siafoundation/renterd:latest-testnet
ghcr.io/siafoundation/hostd:latest
ghcr.io/siafoundation/hostd:latest-testnet
```

should these be updated to reflect `The 'version' should be at the end. e.g.`?

thanks